docs/deprecated.md

```diff
@@ -19,6 +19,16 @@ The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
+### Reserved namespaces in engine labels
+
+**Deprecated in Release: v18.05.0**
```

Comment:
Looks like it was merged after 18.05 was cut, so can you update this to 18.06?

The deprecation policy states we use 3 stable releases (https://docs.docker.com/engine/#installation-guides), so it should become an error in 18.12

Reply:
Done, so sorry for the delay!  Thanks!